src/components/SnippetBlock/index.js

```diff
@@ -9,7 +9,8 @@ const SnippetBlock = ({
   leadingSpaces = 2,
   ...props
 }) => {
-  const raw = children.split(/\r?\n/)
+  // I don't understand why this is needed…
+  const raw = children.toString().split(/\r?\n/)
```

Comment:
`split` is a string method and must be called on a string. React children can be strings, but more often they will be JSX objects, which would have to be converted to a string before splitting it like this.

The mystery for me is how it was working before.

Reply:
btw when copying and pasting code you don't understand, it's best to include a link to the source

Reply:
@lukeg90 I didn't copy and paste, this came from IDE errors, and so I cast it, but much like you I didn't understand why it worked before, so basically I had the same thought process as you but forgot to remove my thought process 😆